cmd/assetsvc/postgresql_utils.go

```diff
@@ -71,7 +71,7 @@ func (m *postgresAssetManager) getPaginatedChartList(namespace, repo string, pag
 	dbQuery := fmt.Sprintf("SELECT info FROM %s %s ORDER BY info ->> 'name' ASC", dbutils.ChartTable, repoQuery)
 	charts, err := m.QueryAllCharts(dbQuery, queryParams...)
 	if err != nil {
-		return nil, 0, nil
+		return nil, 0, err
```

Comment:
:sweat_smile:

Reply:
Yeah, updated tests actually found this :)